build-with-pip.file

```diff
@@ -1,4 +1,10 @@
+
+%if 0%{?pip_name:1}
+%define my_name %{pip_name}
+%else
 %define my_name %(echo %n | cut -f2 -d-)
+%endif
+
```

Comment:
I would have done
```
%if "%{?pip_name:set}" != "set"
%define pip_name  %(echo %n | cut -d- -f2-5)
%endif
```

and use pip_name in the rest of the file (instead of my_name).